tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +153,178 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
-}
 
+  public ReviewerConfig printStartupOsReviewerConfig() throws IOException {
+    String localRegistryFilePath = fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(),
+        "reviewer_config.prototxt");
+    ReviewerConfig reviewerConfig = (ReviewerConfig) fileUtils.readPrototxt(localRegistryFilePath,
+        ReviewerConfig.newBuilder());
+    //System.out.println("StartupOs ReviewerConfig:\n" + reviewerConfig.toString());
+    return reviewerConfig;
+  }
+
+  public ReviewerConfig printHasadnaReviewerConfig() throws IOException {
+    String targetDirectory = fileUtils.expandHomeDirectory("~/hasadna");
+    String localHasadnaRegistryFilePath = fileUtils.joinPaths(targetDirectory, "reviewer_config.prototxt");
+    ReviewerConfig reviewerConfigHasadna = (ReviewerConfig) fileUtils.readPrototxt(localHasadnaRegistryFilePath,
+        ReviewerConfig.newBuilder());
+    //System.out.println("Hasadna ReviewerConfig:\n" + reviewerConfigHasadna.toString());
+    return reviewerConfigHasadna;
+  }
+
+  public void compareReviewerConfigData(ReviewerConfig reviewerConfig1, ReviewerConfig reviewerConfig2) throws IOException {
+    String displayName = reviewerConfig1.getDisplayName();
+    LinkedHashSet<Repo> repoList = new LinkedHashSet<Repo>();
+    //Getting ReviewerConfig1's repos
+    for (Repo repo: reviewerConfig1.getRepoList())
+    {
+      repoList.add(repo);
+    }
+    //Getting ReviewerConfig2's repos
+    for (Repo repo: reviewerConfig2.getRepoList())
+    {
+      repoList.add(repo);
+    }
+    LinkedHashSet<Project> projectList = new LinkedHashSet<Project>();
+    //Getting ReviewerConfig1's projects
+    for (Project project: reviewerConfig1.getProjectList())
+    {
+      projectList.add(project);
+    }
+    //Getting ReviewerConfig2's projects
+    for (Project project: reviewerConfig2.getProjectList())
+    {
+      projectList.add(project);
+    }
+    LinkedHashSet<User> mergedUsersList = new LinkedHashSet<User>();
+    //Getting ReviewerConfig1's user count
+    int reviewerConfig1UserCount = reviewerConfig1.getUserCount();
+    //Getting ReviewerConfig2's user count
+    int reviewerConfig2UserCount = reviewerConfig2.getUserCount();
+    for (int i = 0; i < reviewerConfig1UserCount; i++)
+    {
```

Comment:
Please place open braces at the same line as loop definition. We use this style in StartupOS.

Also for `if` statements.

Reply:
Fixed by using fix_formatting tool.